rhel/vex/parser.go

```diff
@@ -501,11 +546,32 @@ func cvssVectorFromScore(sc *csaf.Score) (vec string, err error) {
 
 // CreatePackageKey creates a unique key to describe an arch agnostic
 // package for deduplication purposes.
-// i.e. AppStream-8.2.0.Z.TUS:python3-idle-0:3.6.8-24.el8_2.2
-func createPackageKey(repo, name, fixedIn string) string {
+// i.e. AppStream-8.2.0.Z.TUS:a_module:python3-idle-0:3.6.8-24.el8_2.2
+func createPackageKey(repo, mod, name, fixedIn string) string {
 	// The other option here is just to use repo + PURL string
 	// w/o the qualifiers I suppose instead of repo + NEVR.
-	return repo + ":" + name + "-" + fixedIn
+	return repo + ":" + mod + ":" + name + "-" + fixedIn
+}
+
+func createPackageModule(p *csaf.Product) (string, error) {
+	var modName string
```

Comment:
This may not be needed anymore

Reply:
won't block the PR, but this could be defined directly above the switch and the return directly below this can just be `""` to match the other scenarios